lib/diff.ml

```diff
@@ -1,39 +1,271 @@
 open Types
 open Stddiff
+open Intermed
```

Comment:
Let's not stack up global opens. This one is only defining types, we can easily go around using the fully qualified names by using type annotations where it matters!